Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -88,16 +101,20 @@ public class NavigationSettings {
      fall back to the `NavigationSettings.directions` by default.
        - tileStoreConfiguration: Options for configuring how map and navigation tiles are stored on the device. See
      `TileStoreConfiguration` for more details.
+     - routingProviderSource: Configures the type of routing to be used by various SDK objects when providing route calculations.
```

Comment:
(This change is mentioned in the changelog blurb being added in #3754.)

Reply:
True, but without #3754 this change is unused. I've added it in this PR though to avoid conflicts between RerouteController and Alternative routes integrations.